code/controllers/mc/controller.dm

```diff
@@ -1,4 +1,19 @@
 /datum/controller
 	var/name
 	// The object used for the clickable stat() button.
-	var/obj/effect/statclick/statclick
\ No newline at end of file
+	var/obj/effect/statclick/statclick
+
+/datum/controller/proc/Initialize()
```

Comment:
TG and similar moved these procs to the controller level years ago but we didn't, this doesn't actually accomplish anything, I think you can revert it